libs/full/compute_cuda/examples/data_copy.cu

```diff
@@ -45,11 +45,11 @@ int hpx_main(hpx::program_options::variables_map& vm)
 
     // copy data from host to device
     hpx::parallel::copy(
```

Comment:
Isn't this supposed to be `hpx::copy` now?

Reply:
Yep, good point. I still have to do a round of checks on the cuda builds.